yaeger.spacesimulator/src/main/java/com/yaeger/spacesimulator/data/ObjectPlacementData.java

```diff
@@ -0,0 +1,80 @@
+package com.yaeger.spacesimulator.data;
+
+import com.github.hanyaeger.api.Coordinate2D;
+
+import javafx.scene.paint.Color;
+
+public class ObjectPlacementData {
+	private double velocity = 1, density = 1, volume = 10;
+	private Coordinate2D startPosition, stopPosition;
+	private boolean placing;
+	private Color color;
+	
+	public ObjectPlacementData() {
+		this.startPosition = new Coordinate2D();
+		this.stopPosition = new Coordinate2D();
+		this.color = new Color(1, 1, 1, 1);
+	}
+	
```

Comment:
Even kijken wat handig is voor ons. Of we dit model initialiseren d.m.v. controlpanel en dat ik een gevuld object met volume, densiteit, diameter en kleur meegeef die verder gevuld kan worden met data van het plaatsen, of dat we dat dus zo doen.

Reply:
Ja, inderdaad.
Als we deze twee dingen willen gaan mergen moeten we hier even voor gaan zitten